examples/run_examples.py

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/python
+#
+# examples/run_examples.py -- run some examples.
+#
+#	Copyright (C) 2008 David Bronke <whitelynx@gmail.com>
+#	Copyright (C) 2002 Peter Liljenberg <petli@ctrl-c.liu.se>
+#
+#	This program is free software; you can redistribute it and/or modify
+#	it under the terms of the GNU General Public License as published by
+#	the Free Software Foundation; either version 2 of the License, or
+#	(at your option) any later version.
+#
+#	This program is distributed in the hope that it will be useful,
+#	but WITHOUT ANY WARRANTY; without even the implied warranty of
+#	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#	GNU General Public License for more details.
+#
+#	You should have received a copy of the GNU General Public License
+#	along with this program; if not, write to the Free Software
+#	Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
+
+import subprocess, unittest
+from subprocess import Popen, PIPE, call
+
+def run(name):
+	proc = Popen("./" + name, shell=True, stdout=PIPE, stderr=PIPE)
+	return proc.wait() == 0
+
+def run_with_args(name, args):
+	for arg in args:
+		proc = Popen("./" + name + " " + arg, shell=True, stdout=PIPE, stderr=PIPE)
+		if proc.wait() != 0:
```

Comment:
Why not just `subprocess.call(...)`? It waits for the process termination as well as `proc.wait()`.